Gemfile.lock

```diff
@@ -132,7 +132,7 @@ GEM
       activesupport (= 5.2.2)
       bundler (>= 1.3.0)
       railties (= 5.2.2)
-      sprockets-rails (>= 2.0.0)
+      sprockets-rails (>= 2.3.2)
```

Comment:
I think something updated here is what is breaking the build on Travis. It doesn't look like you added anything to the gemfile, so there shouldn't be any need to push this. Normally the only thing you edit is the gemfile and the gemfile.lock then gets updated automatically with extra details on the package versions. The best thing for you to do if it's out of sync like is just force pull the gemfile.lock file so it's in sync with master (or you can copy and paste the version that's on github).